File: temp.js
/*
    @author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
    Description of problem at: https://www.codeeval.com/open_challenges/###/
*/

var fs = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {
        line = line.replace(/\r/gm, ""); // For Windows.
        var arr = line.split(';');
		console.log(arr);
		//process.stdout.write();
    }
});

File: temp.c
/*
	@author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
	Description of problem at: https://www.codeeval.com/open_challenges/###/
*/

#include <stdio.h>

int main(int argc, char **argv) {
	FILE *ifp = fopen(argv[1], "r");
	int i;
	while(fscanf(ifp, "%d \n", &i) != EOF) {
		printf("%d ", i);
	}
	printf("\n");
	
	return 0;
}
